ngraph/test/backend/experimental_detectron_roi_feature.in.cpp

```diff
@@ -0,0 +1,124 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+// clang-format off
+#ifdef ${BACKEND_NAME}_FLOAT_TOLERANCE_BITS
+#define DEFAULT_FLOAT_TOLERANCE_BITS ${BACKEND_NAME}_FLOAT_TOLERANCE_BITS
+#endif
+
+#ifdef ${BACKEND_NAME}_DOUBLE_TOLERANCE_BITS
+#define DEFAULT_DOUBLE_TOLERANCE_BITS ${BACKEND_NAME}_DOUBLE_TOLERANCE_BITS
+#endif
+// clang-format on
+#include <numeric>
```

Comment:
I propose to don't add new backend tests. Please extend template_plugin reference tests instead.
https://github.com/openvinotoolkit/openvino/tree/master/docs/template_plugin/tests/functional/op_reference

Reply:
These tests have moved to the template plugin.